# request
time timestamp
---
# response
geometry_msgs/PoseStamped pose
